templates/dex-k8s-authenticator.yaml

```diff
@@ -20,8 +20,8 @@ spec:
     - name: none
       enabled: true
   requires:
-    matchLabels:
-      kubeaddons.mesosphere.io/name: dex
+    - matchLabels:
```

Comment:
This also depends on traefik (ingresscontroller)

Reply:
done.